example_credentials.py

```diff
@@ -23,14 +23,14 @@
 combined_cal = ""
 
 personal_cals = {
-    "Link": "Name1",
-    "Link": "Name_Two",
+    "Link": "Name",
+    "Link_Two": "Name_Two",
 }
 
 quote_url = "https://zenquotes.io/api/today"
 
-
+menu_sheet = ""
```

Comment:
```suggestion
menu_sheet = ""

```

Reply:
What was the change here, I think I'm missing something